src/main/java/com/ipiecoles/java/java350/model/Employe.java

```diff
@@ -59,19 +62,31 @@ public Integer getNbRtt(){
         return getNbRtt(LocalDate.now());
     }
 
-    public Integer getNbRtt(LocalDate d){
-        int i1 = d.isLeapYear() ? 365 : 366;int var = 104;
-        switch (LocalDate.of(d.getYear(),1,1).getDayOfWeek()){
-        case THURSDAY: if(d.isLeapYear()) var =  var + 1; break;
-        case FRIDAY:
-        if(d.isLeapYear()) var =  var + 2;
-        else var =  var + 1;
-case SATURDAY:var = var + 1;
-                    break;
+    public Integer getNbRtt(LocalDate dateReference) {
+        int nbJourAnnee = dateReference.isLeapYear() ? 366 : 365;
+        int nbSamediDimanche = 104;
+
+        // on regarde le  la jour de l'année pour ajouté un jour de week .
+        switch (LocalDate.of(dateReference.getYear(), 1, 1).getDayOfWeek()) {
+            case THURSDAY:
+                if (dateReference.isLeapYear()) nbSamediDimanche = nbSamediDimanche + 1;
```

Comment:
Tu peux même ajouter les accolades, c'est recommandé et plus lisible
```suggestion
                if (dateReference.isLeapYear()) {
                    nbSamediDimanche = nbSamediDimanche + 1;
                }
```